src/plasmapy/particles/particle_class.py

```diff
@@ -2283,6 +2283,30 @@ def __repr__(self) -> str:
             else f"CustomParticle(mass={self.mass}, charge={self.charge}, symbol={self.symbol})"
         )
 
+    def __str__(self) -> str:
+        """
+        Return the particle's symbol if provided, otherwise the |repr|.
```

Comment:
```suggestion
        Return the particle's symbol if provided, otherwise the `repr`.
```
This should probably work, but if not, then we can change it to double back ticks.